_episodes/|1-06-issues.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Using GitHub Issues"
+teaching: 15
+exercises: 10
+questions:
+- "How can I use GitHub issues to manage a TODO list for my project?"
+objectives:
+- "Explain what GitHub issues are and how to create them."
+- "Explain how issues can help you keep track of your work in GitHub"
+- "How to claim issues."
+- "Learn how to use GitHub functionality to effectively communicate who is working on what."
+keypoints:
+- "Issues are a feature of GitHub which let you track work in a respository."
+- "GitHub provides functionality for referencing issues in comments"
+- "Task lists can be created to keep track of a list of issues"
+- "Formatting syntaxes, templates, and subscribing to issues help with communication efficiency"
```

Comment:
```suggestion
- "Formatting syntaxes, templates, and subscribing to issues help with communication"
```

Not sure what's meant by "formatting syntaxes"...?

Reply:
Now you mention it "formatting syntaxes" is not a particularly helpful phrase. Made more sense when I was writing it. Seems that I read and "borrowed" it from here: [Basic writing and formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).